lib/portlayer/vspc/handlers.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
```

Comment:
Are these entirely new files? If so, they should all have just the 2017 copyright.